groupofcells/groupofcells.go

```diff
@@ -49,26 +55,37 @@ func createGroupOfCellsFromArray(arr []uint8, groupOfCells *GroupOfCells, index
 	}
 }
 
-func (gpc GroupOfCells) String() string{
+func (gpc GroupOfCells) String() string {
 	var output string
 	pointer := gpc.farLeftCell
 	for pointer.right != nil {
-		output += fmt.Sprintf("%v,",pointer.value)
+		output += fmt.Sprintf("%v,", pointer.value)
 		pointer = pointer.right
 	}
-	output += fmt.Sprintf("%v",pointer.value)
+	output += fmt.Sprintf("%v", pointer.value)
 
 	return output
 }
 
+func (gpc GroupOfCells) ToSlice() []uint8 {
+	cells := make([]uint8, 0)
+	cell := gpc.farLeftCell
+	for cell.right != nil {
+		cells = append(cells, cell.value)
+		cell = cell.right
+	}
+	cells = append(cells, cell.value)
+	return cells
+}
+
```

Comment:
This is the only other code change I did -- adding a function to turn our linked list back into a `[]uint8`.

Every other change is whitespace only and was done by `gofmt`.